index.js

```diff
@@ -61,16 +67,20 @@ var showQuestions = function() {
     questionsPageElement.show();
 };
 
-var resetScore = function() {
-    scoreElement.text(0);
+var resetState = function(state, element) {
```

Comment:
The critical piece of the exercise is having two different function categories: *state modification functions* and *render functions*.  The first type should ONLY modify the state.  The second type should ONLY read the state and then change the DOM using the element(s) passed in.  This function is disobeying those conditions by both modifying state and changing the DOM.